AuthenticatorPro/Source/Data/Authenticator.cs

```diff
@@ -332,6 +334,7 @@ public static bool IsValidSecret(string secret, AuthenticatorType type)
             {
                 case AuthenticatorType.Totp:
                 case AuthenticatorType.Hotp:
+                case AuthenticatorType.SteamOtp:
```

Comment:
This could be replaced with a type.IsHmacBased() check too. Don't worry about it, I will sort it out.